packages/api-docs-builder/package.json

```
@@ -9,13 +9,13 @@
   "dependencies": {
     "@babel/core": "^7.21.4",
     "@mui-internal/docs-utilities": "^1.0.0",
-    "@mui/markdown": "^5.0.0",
+    "@mui/markdown": "^5.0.1",
     "ast-types": "^0.14.2",
     "doctrine": "^3.0.0",
     "lodash": "^4.17.21",
     "react-docgen": "^5.4.3",
     "recast": "^0.23.2",
-    "typescript-to-proptypes": "^5.0.0",
+    "typescript-to-proptypes": "^5.0.1",
```

```suggestion
    "typescript-to-proptypes": "^5.0.0",
```
No need to increment internal packages. In the release prompt, you can choose "custom version" and then type "5.0.0" manually.